pkg/util/shell/shell.go

```diff
@@ -74,7 +74,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 	switch userShell {
 	case "fish":
 		prefix = "set -gx "
-		suffix = "\";\n"
+		suffix = ";\n"
```

Comment:
@hferentschik as far as I'm aware both would work, but the fish docs themselves don't use quotes (https://fishshell.com/docs/current/tutorial.html#tut_exports), and I never have when adding things to the $PATH in fish so didn't consider adding one :smile: